exist-core/src/main/java/org/exist/backup/restore/RestoreHandler.java

```diff
@@ -405,7 +407,11 @@ private void restoreSubCollectionEntry(final Attributes atts) throws SAXExceptio
         //parse the sub-collection descriptor and restore
         final BackupDescriptor subDescriptor = descriptor.getChildBackupDescriptor(name);
         if(subDescriptor != null) {
-
+            if (pathsToIgnore != null && pathsToIgnore.contains(subDescriptor.getSymbolicPath())) {
+                listener.info("Skipping app path " + subDescriptor.getSymbolicPath() + ". Newer version " +
```

Comment:
I think the message could be more generic, as this class is only concerned with skipping collections, and does not need to know about 'apps', maybe just a simple:

`listener.info("Automatically skipping path: " + subDescriptor.getSymbolicPath());`

Reply:
I guess for users it is important to know why the path was skipped - thus the verbose message.